app/src/lib/stores/app-store.ts

```diff
@@ -1350,7 +1372,7 @@ export class AppStore extends TypedBaseStore<IAppState> {
       repository,
       account,
       r => this.performFetch(r, account, FetchType.BackgroundTask),
-      r => this.shouldBackgroundFetch(r)
+      r => this.shouldBackgroundFetch(r, null)
```

Comment:
Can we add a `TODO` in here to follow up on this? I think we should make the background fetcher follow the same rules as refreshing the indicators.

Reply:
Are you referring to the limits we put in place? or checking the API before fetching?

Reply:
Checking the API before fetching